Automattic/jetpack
# Boost: rewrite SuperCacheInfo component in React

Rewrite the Svelte component, SuperCacheInfo in React

## Proposed changes:
* The SuperCacheInfo component is changed from Svelte to React


### Other information:


## Jetpack product discussion
pc9hqz-2jy-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Apply PR
* run `npm run dev` in projects/plugins/boost and wait for changes to compile
* Load Boost settings page and confirm the "WP Super Cache" notice does not appear at the ned of the page.
* Install WP Super Cache and activate it, but do not enable caching.
* Check for "Super Cache is installed but not enabled" notice on Boost settings page.
* Click on "Set up" button to go to WP Super Cache settings page and enable caching.
* Reload Boost settings page and confirm the notice now says "Super Cache detected".
* Click on "Run test" button to run a test and check that it shows a loading button and then returns a result.
* To test the error notice, add `setState( { status: 'error', error: 'testing the error' } );` at the end of the `runTest` function.
* Reload and run the test again and you will see the error notice, which includes a "Re-run test" button.
* Disable caching, reload Boost settings page and click X on the "Super Cache is installed but not enabled" notice to close it.
* Reload Boost settings page and check that notice does not appear again.
